core/plugins/jdo/datanucleus-5/src/test/java/org/apache/isis/runtime/persistence/objectstore/transaction/PojoAdapterBuilder.java

```diff
@@ -160,7 +151,6 @@ public PojoAdapterBuilder with(Type type) {
     }
 
     public PojoAdapterBuilder with(ObjectSpecification objectSpec) {
-        this.objectSpec = objectSpec;
```

Comment:
was it intentional to remove this line?  If so, what purpose does the method now have?

Reply:
unused fields removed, but didn't remove unused methods, fixing that